tests/test_ssl_ecdh_curve.rb

```diff
@@ -50,7 +52,7 @@ def ssl_handshake_completed
   def test_no_ecdh_curve
     omit("No SSL") unless EM.ssl?
     omit_if(rbx?)
-    omit("OpenSSL 1.1.x (and later) auto selects curve") if OPENSSL_GT_1_0
+    omit("OpenSSL 1.1.x (and later) auto selects curve") if SSL_LIB_VERS >= '1.1'
```

Comment:
FYI - Nit - This is just a text comparison, not a formal version comparison. It's not a problem, just wanted to point it out.

Reply:
Using an array now